### PR TITLE
Readme linter corrections (second run)

### PR DESCRIPTION
**Pull request by Confectrician**
According to the comment of @ThomDietrich in #2948 -> https://github.com/openhab/openhab2-addons/pull/2948#issuecomment-351656082

There were still some linter errors present in several binding files.
After digging into this and finally settng up my editor and remark config this is the next try, to get rid of those errors and warnings finally.

I would appreciate a deep review from @ThomDietrich and a feedback if my linter tools are setup up correctly according to the wanted behavour and style for the documentation.

Addationally we should check if the rules are configured correctly too.
Have a look at the changes made to the atlona readme within this pr.
For one list the linter tool told me to remove spaces in lists and for another lists with more than one line per list item the linter tool told me to add spaces and newlines.
Here i would apreciate some feedback from @ThomDietrich too.

If my settings are confirmed and the linting validation is fine, i will add some more readme's to this PR.

### As already done this way
Those changes are only linter related. No content was or will be imrpoved in this PR.
Of course small things like correct markdown tables are ok and will be changed too.

Signed-off-by Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

File: addons/binding/org.openhab.binding.allplay/README.md [hunk 1]
```diff
@@ -108,22 +103,22 @@ String All2CoverUrl  "Cover Art URL [%s]"   {channel="allplay:speaker:9fbe37ca-d
...
-			Text item=All2State
-		}
+    Frame label="All2" {
+      Default item=All2Control
+      Slider item=All2Volume
+      Text item=All2Title
+      Text item=All2Artist
+      Text item=All2State
+    }
 }
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.allplay/README.md]: Indention isn't right here. Please move the closing bracket one line up, then use for spaces per indention level.

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 2]
```diff
@@ -78,6 +78,7 @@ Follow the instructions to set up the Dash Button in their mobile app.
 When you get to the step where it asks you to pick which product you want to map it to, just quit the setup process.
 
 {: #no-phonehome}
+
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: No. This is an exception we should teach the linter by submitting a PR.
This is a "block inline attribute" defined for Kramdown https://kramdown.gettalong.org/quickref.html#block-attributes
I am almost certain that it has to be next to the headline element without an empty line.

File: addons/binding/org.openhab.binding.atlona/README.md [hunk 3]
```diff
@@ -21,20 +21,21 @@ If "Telnet Login" is enabled ("Network" tab from the switch configuration UI), y
 
 ## Binding configuration
 
-```
+```text
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.atlona/README.md]: java?
```java
atlona:pro3-88m:home [ipAddress="192.168.1.30", userName="me", password="12345", polling=600, ping=30, retryPolling=10]
```

File: addons/binding/org.openhab.binding.atlona/README.md [hunk 4]
```diff
@@ -179,44 +177,47 @@ The # of presets allowed depends on the firmware you are using (5 presets up to
...
+- If telnet mode is enabled, you must use the admin username/password to issue a matrixreset
 
 ## Example
 
 ### Things
 
 Here is an example with minimal configuration parameters (using default values with no telnet login):
 
-```
+```text
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.atlona/README.md]: java?

**Comment by Confectrician** [comment]
Ok i have adressed the comments and will now go on with some more bindings and do a second check there.

@ThomDietrich 
> Thanks for taking up this tedious work!

You're welcome.
I am really annoyed of all those linter issues which are lying around the docs repo for months now and simply hadn't a running setup for the linter tool yet.

Time to get rid of this stuff or at least 90% of it, since we won't get all warnings solved without contributing to remark for now.

**Comment by Confectrician** [comment]
I hav added all bindings until **H**.
I think that are enough changes for one PR.

@ThomDietrich and @martinvw -> ready for reaview.
I will go on with other bindings later.

About the conflict: Seems that a table has been updated in th echromecast readme and the current on in master should be used.

File: addons/binding/org.openhab.binding.airquality/README.md [hunk 5]
```diff
@@ -92,15 +87,15 @@ HAZARDOUS=Hazardous
 
 airquality.things:
 
-```
+```text
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.airquality/README.md]: Why are all of those "text"? I'd suggest anything other than text to get a colored highlighting. As with most openHAB DSL text, they look good with a "java" setting:

```javascript
 airquality:aqi:home "AirQuality" @ "Krakow" [ apikey="XXXXXXXXXXXX", location="50.06465,19.94498", refresh=60 ]
 airquality:aqi:warsaw "AirQuality in Warsaw" [ apikey="XXXXXXXXXXXX", location="52.22,21.01", refresh=60 ]
 airquality:aqi:brisbane "AirQuality in Brisbane" [ apikey="XXXXXXXXXXXX", stationId=5115 ]
```

File: addons/binding/org.openhab.binding.allplay/README.md [hunk 6]
```diff
@@ -90,13 +85,13 @@ Audio streams are sent to the `stream` channel.
 
 demo.things:
 
-```
+```text
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.allplay/README.md]: java

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 7]
```diff
@@ -112,17 +114,19 @@ You can ignore these devices in your Inbox.
...
-*   `macAddress` - The MAC address of the Amazon Dash Button.
-*   `pcapNetworkInterfaceName` - The network interface which receives the packets of the Amazon Dash Button.
-*   `packetInterval` - Often a single button press is recognized multiple times.
+-   `macAddress` - The MAC address of the Amazon Dash Button.
+
+-   `pcapNetworkInterfaceName` - The network interface which receives the packets of the Amazon Dash Button.
+
+-   `packetInterval` - Often a single button press is recognized multiple times.
 You can specify how long any further detected button pressed should be ignored after one click was processed.
 The parameter is optional and 5000ms by default.
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: Shouldn't these two lines be indented?

File: addons/binding/org.openhab.binding.atlona/README.md [hunk 8]
```diff
@@ -21,20 +21,21 @@ If "Telnet Login" is enabled ("Network" tab from the switch configuration UI), y
...
--   password: (optional) the password to login with (only if Telnet Login is enabled)
--   polling: (optional) the time (in seconds) to poll the state from the actual switch (default: 600)
--   ping: (optional) the time (in seconds) to ping the switch to keep our connection alive (default: 30)
--   retryPolling: (optional) the time (in seconds) to retry a connection if the connection has failed (default: 10)
+- ipAddress: Hostname or IP address of the matrix switch
+- userName: (optional) the username to login with (only if Telnet Login is enabled)
+- password: (optional) the password to login with (only if Telnet Login is enabled)
+- polling: (optional) the time (in seconds) to poll the state from the actual switch (default: 600)
+- ping: (optional) the time (in seconds) to ping the switch to keep our connection alive (default: 30)
+- retryPolling: (optional) the time (in seconds) to retry a connection if the connection has failed (default: 10)
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.atlona/README.md]: As in most other binding READMEs, please put the parameters in:

> `monospace` - Description

File: addons/binding/org.openhab.binding.avmfritz/README.md [hunk 9]
```diff
@@ -125,23 +122,23 @@ Switch COMETDECTBattery "Battery low" (gCOMETDECT) { channel="avmfritz:Comet_DEC
...
+  }
+  Frame label="FRITZ!Powerline 546E switchable outlet" {
+    Switch item=Outlet2
+  }
+  Frame "Comet DECT heating thermostat" {
+    Text item=COMETDECTActualTemp icon="temperature"
+    Setpoint item=COMETDECTSetTemp minValue=8.0 maxValue=28.0 step=0.5 icon="temperature"
+    Selection item=COMETDECTRadiatorMode mappings=["ON"="ON", "OFF"="OFF", "COMFORT"="COMFORT", "ECO"="ECO", "BOOST"="BOOST"]
+    Switch item=COMETDECTBattery icon="battery"
+  }
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.avmfritz/README.md]: Indent with four spaces?

File: addons/binding/org.openhab.binding.coolmasternet/README.md [hunk 10]
```diff
@@ -23,43 +22,18 @@ Bridge coolmasternet:controller:main [ host="192.168.0.100" ] {
...
+| on           | Switch    | Turn HVAC unit on and off.                                                                                                                       |
+| mode         | String    | HVAC Mode. "Auto", "Cool", "Heat", "Dry" or "Fan Only". Unit may not support all modes.                                                          |
+| fan          | String    | Fan Mode. "Auto", "Low", "Medium", "High" or "Top". Unit may not support all speeds.                                                             |
+| set_temp     | Number    | Temperature target setpoint.                                                                                                                     |
+| current_temp | Number    | Current temperature at HVAC unit.                                                                                                                |
+| louvre       | String    | Louvre angle. "No Control", "Auto Swing", "Horizontal", "30 degrees", "45 degrees", "60 degrees" or "Vertical". Unit may not support all angles. |
```
Comment (ThomDietrich) [inline addons/binding/org.openhab.binding.coolmasternet/README.md]: THANK YOU 😄

**Comment by Confectrician** [comment]
I didn't check the "already commited files" since you reviewed.
Now everything of the "A Binding Readmes" should have been done too.

File: addons/binding/org.openhab.binding.coolmasternet/README.md (hunk 10 above)
Comment (Confectrician) [reply addons/binding/org.openhab.binding.coolmasternet/README.md]: I have fixed "direct html tables" wherever possible.
But i didn't check how to handle nested tables in markdown, so i didn't touch those.

**Comment by Confectrician** [comment]
We can't observe future changes due to other PRs as well all the time.
So getting rid of a greater percentage of these issues is perfectly fine for me.

Every issue we can fix now, won't get used for copy paste readme's in the future. 😄

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 11]
```diff
@@ -99,7 +99,9 @@ There is one supported Thing, the "Amazon Dash Button".
 
 ## Discovery
 
-Background discovery is not supported as it is not possible to distinguish between a Dash Button and other Amazon devices like the Kindle, a Fire TV or an Echo speaker.
+Background discovery is not supported as it is not possible to distinguish
+between a Dash Button and other Amazon devices like the Kindle,
+a Fire TV or an Echo speaker.
```
Comment (bgilmer77) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: Consider changing to "Background discovery is not supported, as it is not possible to distinguish between a Dash Button and some other type of Amazon devices such as the Kindle reader, a Fire TV, or an Echo speaker."

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 12]
```diff
@@ -112,17 +114,19 @@ You can ignore these devices in your Inbox.
...
-*   `pcapNetworkInterfaceName` - The network interface which receives the packets of the Amazon Dash Button.
-*   `packetInterval` - Often a single button press is recognized multiple times.
-You can specify how long any further detected button pressed should be ignored after one click was processed.
-The parameter is optional and 5000ms by default.
+-   `macAddress` - The MAC address of the Amazon Dash Button.
+
+-   `pcapNetworkInterfaceName` - The network interface which receives the packets of the Amazon Dash Button.
+
+-   `packetInterval` - Often a single button press is recognized multiple times.
+    You can specify how long any further detected button pressed should be ignored after one click was processed.
```
Comment (bgilmer77) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: Consider changing to, "You can specify a period of time, after an initial button-press, additional button-presses are ignored."

File: addons/binding/org.openhab.binding.amazondashbutton/README.md [hunk 13]
```diff
@@ -112,17 +114,19 @@ You can ignore these devices in your Inbox.
...
+    The parameter is optional and 5000ms by default.
```
Comment (bgilmer77) [inline addons/binding/org.openhab.binding.amazondashbutton/README.md]: Consider changing to, "This parameter is optional, and is set to 5000ms by default."

File: addons/binding/org.openhab.binding.atlona/README.md [hunk 14]
```diff
@@ -21,20 +21,21 @@ If "Telnet Login" is enabled ("Network" tab from the switch configuration UI), y
...
+- `password`: (optional) the password to login with (only if Telnet Login is enabled)
+- `polling`: (optional) the time (in seconds) to poll the state from the actual switch (default: 600)
+- `ping`: (optional) the time (in seconds) to ping the switch to keep our connection alive (default: 30)
+- `retryPolling`: (optional) the time (in seconds) to retry a connection if the connection has failed (default: 10)
 
 ### username/password
 
-The userName/password configuration options are optional and are only required if you have your switch set with "Telnet Login" enabled (on the "Network" tab from the switch configuration UI).  The user must be a valid user listed on the "Users" tab of the switch configuration UI in this case.
+The userName/password configuration options are optional and are only required if you have your switch set with "Telnet Login" enabled (on the "Network" tab from the switch configuration UI).
+The user must be a valid user listed on the "Users" tab of the switch configuration UI in this case.
```
Comment (bgilmer77) [inline addons/binding/org.openhab.binding.atlona/README.md]: Consider changing to, "`userNAME` must be a valid user listed on the "Users" tab of..."

File: addons/binding/org.openhab.binding.atlona/README.md [hunk 15]
```diff
@@ -179,44 +177,47 @@ The # of presets allowed depends on the firmware you are using (5 presets up to
 As of firmware 1.6.03 (rev 13), there are three issues on Atlona firmware (I've notified them on these issues):
 
 -   clearX command does not work.  The TCP/IP command "ClearX" as specified in Atlona's protocol will ALWAYS return a "Command Failed".  Please avoid this channel until atlona releases a new firmware.
--   There is no way to query what the current status is of: panellock, and irenable.  This addon simply assumes that panellock is off and irenable is on at startup.  
--   If you make a change in the switches UI that requires a reboot (mainly changing any of the settings on the "Network" tab in the switch configuration UI), this addon's connection will be inconsistently closed at different times.  The thing will go OFFLINE and then back ONLINE when the reconnect attempt is made - and then it starts all over again.  Please make sure you reboot as soon as possible when the switch UI notifies you.
+
+-   There is no way to query what the current status is of: panellock, and irenable.  This addon simply assumes that panellock is off and irenable is on at startup.
```
Comment (bgilmer77) [inline addons/binding/org.openhab.binding.atlona/README.md]: Consider changing to, "There is no way to query the current status of: panellock and..."

Also consider moving the next sentence starting, "This addon simply..." to its own line.

File: addons/binding/org.openhab.binding.dscalarm/README.md [hunk 16]
```diff
@@ -50,27 +56,28 @@ DSC Alarm things can be configured either through the online configuration utili
...
-	Thing zone zone14 [ partitionNumber=1, zoneNumber=14 ]
-	Thing zone zone15 [ partitionNumber=1, zoneNumber=15 ]
-	Thing zone zone21 [ partitionNumber=1, zoneNumber=21 ]
-	Thing zone zone22 [ partitionNumber=1, zoneNumber=22 ]
-	Thing zone zone23 [ partitionNumber=1, zoneNumber=23 ]
-	Thing zone zone24 [ partitionNumber=1, zoneNumber=24 ]
-	Thing zone zone25 [ partitionNumber=1, zoneNumber=25 ]
-	Thing keypad keypad
+The binding can be configured manually if discovery is not used.  
+A thing configuration file in the format 'bindingName.things' would need to be created, and placed in the 'conf/things' folder.
```
Comment (bgilmer77) [inline addons/binding/org.openhab.binding.dscalarm/README.md]: Consider changing to, "and placed in the `$OPENHAB_CONF/things` folder."

File: addons/binding/org.openhab.binding.dscalarm/README.md [hunk 17]
```diff
@@ -50,27 +56,28 @@ DSC Alarm things can be configured either through the online configuration utili
...
-	Thing zone zone15 [ partitionNumber=1, zoneNumber=15 ]
-	Thing zone zone21 [ partitionNumber=1, zoneNumber=21 ]
-	Thing zone zone22 [ partitionNumber=1, zoneNumber=22 ]
-	Thing zone zone23 [ partitionNumber=1, zoneNumber=23 ]
-	Thing zone zone24 [ partitionNumber=1, zoneNumber=24 ]
-	Thing zone zone25 [ partitionNumber=1, zoneNumber=25 ]
-	Thing keypad keypad
+The binding can be configured manually if discovery is not used.  
+A thing configuration file in the format 'bindingName.things' would need to be created, and placed in the 'conf/things' folder.  
+Here is an example of a thing configuration file called 'dscalarm.things':
```
Comment (bgilmer77) [inline addons/binding/org.openhab.binding.dscalarm/README.md]: Consider changing to, "example of a Thing...". (Capital "T")

File: addons/binding/org.openhab.binding.hdanywhere/README.md [hunk 18]
```diff
@@ -4,16 +4,18 @@ This binding integrates with [HDanywhere](http://www.hdanywhere.co.uk) HDMI matr
 
 HDanywhere is a manufacturer of multiroom/distributed audio/video equipment.
 This binding supports their V3 of the Multiroom+ HDMI matrix running firmware V1.2(20131222), as well as the newer MHUB series of matrices.
-These matrices support the highest HD resolutions, including 1080p 3D & 4K, use a single Cat5e/6/7 wiring structure with reliable performance up to 100m, have IR passback to allow you to select and control what you watch from every room and are fully compatible with universal remote controls
+These matrices support the highest HD resolutions, including 1080p 3D & 4K, use a single Cat5e/6/7 wiring structure with reliable performance up to 100m,
+have IR passback to allow you to select and control what you watch from every room and are fully compatible with universal remote controls
```
Comment (bgilmer77) [inline addons/binding/org.openhab.binding.hdanywhere/README.md]: This whole section sounds like a commercial for the product.  Delete it?  Minimize the marketing language?

File: addons/binding/org.openhab.binding.hdanywhere/README.md [hunk 19]
```diff
@@ -25,9 +27,10 @@ This binding does not require any special configuration.
 
 ## Thing Configuration
 
-Each thing requires the IP address of the matrix, and the interval in between status updates that are fetched from the matrix. Additionally, the *multiroomplus* has an additional required parameter 'ports' to specify the number of physical ports (e.g. 4x4, 8x8,...) of the matrix.
+Each thing requires the IP address of the matrix, and the interval in between status updates that are fetched from the matrix.
```
Comment (bgilmer77) [inline addons/binding/org.openhab.binding.hdanywhere/README.md]: Consider changing to, "Each Thing..." (Capital "T").

**Comment by bgilmer77** [comment]
Rats - just saw this is a "linter-only" check.  Please disregard all of my comments for now.

**Comment by martinvw** [comment]
Thanks!

**Pull request merged**